docs/roles/system_manager/manage_deployments.md

```diff
@@ -6,6 +6,34 @@
 This document assumes that you already have access to a {ref}`Safe Haven Management (SHM) environment <deploy_shm>` and one or more {ref}`Secure Research Environments (SREs) <deploy_sre>` that are linked to it.
 ```
 
+(resize_vm)=
+
+## {{arrow_upper_right}} Resize the Virtual Machine (VM) of a Secure Research Desktop (SRD)
+
+Sometimes during a project that uses a deployed SRE, researchers may find the available compute inadequate for their purposes and wish to increase the size of the SRD's VM.
+
+- Log into the Azure portal and locate the VM inside the Resource Group called `RG_SHM_<shm id>_SRE_<sre id>_COMPUTE`
+- The simplest way to resize this VM is by [following these instructions](https://learn.microsoft.com/en-us/azure/virtual-machines/resize-vm?tabs=portal) in the Azure portal
+- Alternatively, you can resize this VM by using the below instructions to "Add a new SRD", taking care to set the `<IP last octet>` to that of the VM (viewable in the Azure portal), and by setting the `-Upgrade` and `-Force` flags
```

Comment:
I think it would be clearer to put an example command here,

Like `./Add_Single_SRD.ps1 -shmId <SHM ID> -sreId <SRE ID> -ipLastOctet <IP last octet> [-vmSize <VM size>] -Upgrade -Force` rather than saying 'look at some other instructions and remember to add these flags'.

It might also be worth explaining what `-Upgrade` and `-Force` do and how this is different from changing size in the portal.